src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -524,13 +524,18 @@ public function makePathRelative($endPath, $startPath)
     /**
      * Mirrors a directory to another.
      *
+     * All files and directories in the origin directory will be copied into the target directory. By default:
```

Comment:
> All files and directories[...]
not true if $iterator is provided, isn't it?

Reply:
Right, I've rephrased a bit in #26215 and expanded the documentation of the `$iterator` parameter.